tests/core/test_verifying.py

```diff
@@ -120,4 +120,79 @@ def test_ecr(seeder):
+       
+        qvicred = get_qvi_cred(issuer=hab.pre, recipient=hab.pre, schema=Schema.QVI_SCHEMA, registry=registry)
+        # created verifiable credential.
+        hab, qcrdntler, qsaid, qkmsgs, qtmsgs, qimsgs, qvimsgs = get_cred(hby, hab, regery, registry, verifier, Schema.QVI_SCHEMA, qvicred, seqner)
+        
+        qviedge = get_qvi_edge(qvicred.sad["d"], Schema.QVI_SCHEMA)
+       
+        leicred = get_lei_cred(issuer=hab.pre, recipient=hab.pre, schema=Schema.LEI_SCHEMA, registry=registry, sedge=qviedge)
+        hab, lcrdntler, lsaid, lkmsgs, ltmsgs, limsgs, leimsgs = get_cred(hby, hab, regery, registry, verifier, Schema.LEI_SCHEMA, leicred, seqner)
+
+        #chained ecr auth cred
+        eaedge = get_ecr_auth_edge(lsaid,Schema.LEI_SCHEMA)
+        
+        eacred = get_ecr_auth_cred(aid=hab.pre, issuer=hab.pre, recipient=hab.pre, schema=Schema.ECR_AUTH_SCHEMA, registry=registry, sedge=eaedge)
+        hab, eacrdntler, easaid, eakmsgs, eatmsgs, eaimsgs, eamsgs = get_cred(hby, hab, regery, registry, verifier, Schema.ECR_AUTH_SCHEMA, eacred, seqner)
+        #chained ecr auth cred
+        # ecredge = get_ecr_edge(easaid,Schema.ECR_AUTH_SCHEMA)
+        
+        # ecr = get_ecr_cred(issuer=hab.pre, recipient=hab.pre, schema=Schema.ECR_SCHEMA, registry=registry, sedge=ecredge)
+        # hab, eccrdntler, ecsaid, eckmsgs, ectmsgs, ecimsgs, ecmsgs = get_cred(hby, hab, regery, registry, verifier, Schema.ECR_SCHEMA, ecr, seqner)
+        
+        app = falcon.App()
+        vdb = basing.VerifierBaser(name=hby.name, temp=True)
+        verifying.setup(app=app, hby=hby, vdb=vdb, reger=eacrdntler.rgy.reger)
+
+        issAndCred = bytearray()
+        # issAndCred.extend(kmsgs)
+        # issAndCred.extend(tmsgs)
+        # issAndCred.extend(imsgs)
+        # issAndCred.extend(eamsgs)
+        acdc = issAndCred.decode("utf-8")
+
+        # Create a test client
+        client = falcon.testing.TestClient(app)
+        # Define the said and the credential
+        result = client.simulate_put(f'/presentations/{easaid}',
+                                        body=acdc,
+                                        headers={'Content-Type': 'application/json+cesr'})
+        assert result.status == falcon.HTTP_400
+
+        issAndCred.extend(eamsgs)
+        acdc = issAndCred.decode("utf-8")
+        result = client.simulate_put(f'/presentations/{easaid}',
+                                        body=acdc,
+                                        headers={'Content-Type': 'application/json'})
+        assert result.status == falcon.HTTP_400
+        
+        hby.kevers[hab.pre] = hab.kever
+        
+        auth = Authorizer(hby, vdb, eacrdntler.rgy.reger, [LEI])
+        auth.processPresentations()
+        
+        result = client.simulate_get(f'/authorizations/{hab.pre}')
+        assert result.status == falcon.HTTP_403
+
+        auth_result = client.simulate_get(f'/authorizations/{'EKC8085pwSwzLwUGzh-HrEoFDwZnCJq27bVp5atdMT9'}')
```

Comment:
To signal to other devs that this is meant to be an unknown_prefix could you create a variable with this value (or better yet lets make the value 'bad-id') and then use that variable here and below in the /request/verify post call that is made